src/main.ts

```diff
@@ -457,34 +457,57 @@ class GeneralSettingsTab extends PluginSettingTab {
         new Setting(this.containerEl)
             .setName("Automatic Task Completion Tracking")
             .setDesc(
-                "If enabled, Dataview will automatically append tasks with their completion date when they are checked in Dataview views."
+                "If enabled, Dataview will automatically append tasks with their completion date when they are checked in Dataview views." +
+                    " Example with default field name and date format: - [x] my task [completion:: 2022-01-01]"
             )
             .addToggle(toggle =>
                 toggle.setValue(this.plugin.settings.taskCompletionTracking).onChange(async value => {
                     await this.plugin.updateSettings({ taskCompletionTracking: value });
+                    // Force reload of settings tab
+                    this.display();
                 })
             );
 
-        new Setting(this.containerEl)
-            .setName("Automatic Task Completion Field")
-            .setDesc(
-                "Text used as inline field key to track task completion date when toggling a task's checkbox in a dataview view."
-            )
-            .addText(text =>
-                text.setValue(this.plugin.settings.taskCompletionText).onChange(async value => {
-                    await this.plugin.updateSettings({ taskCompletionText: value.trim() });
-                })
-            );
+        if (this.plugin.settings.taskCompletionTracking) {
+            new Setting(this.containerEl)
+                .setName("Use Emoji Shorthand for Task Completion")
+                .setDesc(
+                    "If enabled, will use emoji shorthand for task completion date instead of inline field formatting. Example: - [x] my task ✅ 2022-01-01" +
+                        " This fills out the implicit Dataview field `completion` for the task with the date." +
+                        " Disable this if you want to customize the completion date format or the field name, or to use Dataview inline field formatting."
```

Comment:
Not sure of the wording, but might it be worth mentioning why they might do this, for compatibility with the Tasks plugin?

Reply:
No sorry, I don't. If it were me, I'd probably use some loose wording like "for example, ..." - since it is at over 100,000 downloads now, it's going to be commonly used.